README.md

```diff
@@ -0,0 +1,88 @@
+# Magic Platformer
+
+Magic platformer is a competitive game to be played with your friends. It is a
+dynamic 2D arcade platformer, where the main objective is to be the last one alive.
+Gameplay revolves around using various spells to deal damage your opponent(s),
+while simultaneously running and jumping on platforms to avoid being hit and
+collect pick-ups.
+
+## Installation
+
+You have two options to install the game. You can either download the latest
+release, or build the game yourself in Unity.
+
+### Downloading the latest release
+To download the latest release, go to the [releases](/releases/latest) page and
+choose the appropriate file for your platform.
+
+**Note:** The releases have been tested only on Windows 10 and Linux (Fedora 38).
+
+### Opening in Unity
+
+To open the project in Unity, you need to have the Unity editor installed.
+Clone the repository, using the following command:
+
+```bash
+git clone https://github.com/Wolokin/MagicPlatformer.git
+```
+
+Then open the project in Unity by selecting the `MagicPlatformer` folder.
+
+## Quick start
+
+### Start menu
+
+After running the game, you will be greeted with the start menu. From here you
+can select the number of players (2-4), the skin of each player and the input
+they will use.<br>
```

Comment:
```suggestion
you will use.<br>
```

Reply:
I disagree, I think 'they' is more appropriate here, since I am referring to each player that will be playing, not just the one setting up the game.

Reply:
Ok, that makes sense